src/expr/src/expr/test_utils.rs

```diff
@@ -46,3 +46,27 @@ pub fn make_input_ref(idx: i32, ret: TypeName) -> ExprNode {
         rex_node: Some(RexNode::InputRef(InputRefExpr { column_idx: idx })),
     }
 }
+
+pub fn make_i32_literal(data: i32) -> ExprNode {
```

Comment:
Consider using `ExprImpl::literal_int(1).to_expr_proto()` to replace this function.